README.md

```diff
@@ -120,7 +120,7 @@
 If you would like to contribute to this repository, please...
 
 1. [![fork](https://user-images.githubusercontent.com/136959/42383736-c4cb0db8-80fd-11e8-91ca-12bae108bccc.png) Fork](https://github.com/StylishThemes/GitHub-Dark/fork)
-1. Make sure you have these installed:
+2. Make sure you have these installed:
```

Comment:
Not strictly a typo because this 1., 1., 1. numbering is a feature of markdown which saves the user from having to count and renumber when adding to the list.

Reply:
Agreed, but in plain text it looks like a typo. _Installation steps_ also use proper numbers.

Reply:
Yeah fine here I guess